tests/testthat/test-calculate_tdm.R

```diff
@@ -51,9 +60,11 @@ test_that("outputs the expected names", {
 test_that("outputs 0 for `tdm_technology_value` and `tdm_sector_value`, if `scen_alloc_wt_tech_prod
           doesn't change over 10 year period`", {
   data <- fake_tdm_data(year = c(2020, 2025, 2030))
-  out <- calculate_tdm(data, 2020)
-  expect_true(out$tdm_technology_value == 0)
-  expect_true(out$tdm_sector_value == 0)
+  out <- calculate_tdm(data, 2020) %>%
+    split(.$tdm_metric)
```

Comment:
Most tests had to gain a line similar to `split(.$tdm_metric)` to account for the change in the structure of the output tibble.